jumpscale/sals/marketplace/apps_chatflow.py

```diff
@@ -144,7 +143,8 @@ def _get_domain(self):
         return self.domain
 
     @chatflow_step(title="Solution Name")
-    def get_solution_name(self):
+    def solution_name(self):
```

Comment:
seems you went back to overriding the step name. make sure to merge the base branch

Reply:
okay, will merge again with `development_capacitypool`
thank you